app/models/timeline_event_file.rb

```diff
@@ -3,6 +3,7 @@ class TimelineEventFile < ApplicationRecord
   has_one_attached :file
 
   validates :file, attached: true
+  validates_with RateLimitValidator, limit: 5, scope: :timeline_event_id
```

Comment:
This is likely to be quite insufficient. `CreateSubmission` mutation's `ValidateStudentSubmission` module checks for a max of 3 files _per_ file question, and there's also audio file uploads. This will probably need a much increased rate limit. 50?